src/install/dotnet/installation/wcf-enable.mdx

```diff
@@ -0,0 +1,98 @@
+---
+componentType: default
+optionType: deployment
+headingText: Enable the agent (optional)
+descriptionText: "" 
+---
+
+For self-hosted WCF applications (hosted as a custom Windows Service, a console app, or a Windows Forms app), you must explicitly enable the agent for your application(s). Here are instructions for .NET Framework and .NET Core: 
+
+<Tabs>
+  <TabsBar>
+    <TabsBarItem id="enable-framework">.NET Framework</TabsBarItem>       
+    <TabsBarItem id="enable-core">.NET Core</TabsBarItem>
+  </TabsBar>
+
+  <TabsPages>
+    <TabsPageItem id="enable-framework">
+
+Enable the agent for your application with one of the following methods:
+
+<CollapserGroup>
```

Comment:
Collapsers withing tabs are a pattern I'd shy away from in general, but I think it's the right call here!